reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md

```diff
@@ -138,18 +153,19 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: False
+Default value: None

```

Comment:
Change all switch parameters default to **False**.

Reply:
I thought I had bulk fixed all of these. 
Fixed